lib/fathead/jmeter/fetch.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
Hey @pjhampton ,
Updated the directory name, still doesnt seem to be picked up :(